modules/sb-start-asb.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * applications/service_brokers/installing-ansible-service-broker.adoc
+
+[id='sb-start-asb-{context}']
+= Starting the {asb-name}
+
+Once you have installed the {asb-name} Operator, you can start the {asb-name} using the following procedure.
+
+.Prerequisites
+
+* You have enabled service catalog.
+* You have installed the {asb-name} Operator.
+
+.Procedure
+
+. Navigate in the web console to *Catalog* -> *Installed Operators* and select the `ansible-service-broker` project.
+. Select the *OpenShift Ansible Service Broker Operator*.
+. Under *Provided APIs*, click *Create New* for *Automation Broker*.
+. Review the default YAML and click *Create*.
+
+Once the {asb-name} has started, you can view the available service bundles by navigating to *Catalog* -> *Developer Catalog* and selecting the *Service Class* checkbox. Note that it may take a few minutes for the {asb-name} to start and the service bundles to be available.
+
+If you do not yet see these Service classes, you can check the status of the following items:
+
+* {asb-name} Pod status
+** From the *Workloads* -> *Pods* page for the *ansible-service-broker* project, verify that the Pod that starts with `asb-` has a status of *Running* and readiness of *Ready*.
+* Service catalog controller manager Pod logs
```

Comment:
Add checking of clusterservicebroker status.
from UI:
Catalog-> Broker Management-> Service Brokers
status of 'ansible-service-broker ' should be ready.

Broker status will be ready a few minutes after the '=apiserver-' ready.